mingw-w64-opencl-icd/PKGBUILD

```
@@ -1,38 +1,25 @@
 # Maintainer: Peter Budai <peterbudai at hotmail.com>
 
 _realname=opencl-icd
-pkgbase=mingw-w64-${_realname}-git
-pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
-provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
-pkgver=47.c7fda8b
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
```

```suggestion
pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
replaces=("${MINGW_PACKAGE_PREFIX}-${_realname}-git")
```

otherwise updates are broken if opencl-icd-git is installed already